tests/kernel/device/src/test_driver_init.c

```diff
@@ -150,6 +150,10 @@ DEVICE_DEFINE(my_driver_level_4, MY_DRIVER_LV_4, &my_driver_lv_4_init,
 		device_pm_control_nop, NULL, NULL, APPLICATION,
 		CONFIG_KERNEL_INIT_PRIORITY_DEFAULT, &funcs_my_drivers);
 
+DEVICE_DEFINE(my_driver_priority_4, MY_DRIVER_PRI_4,
+		&my_driver_pri_4_init, device_pm_control_nop,
+		NULL, NULL, POST_KERNEL, 20, &funcs_my_drivers);
```

Comment:
Probably it worth to leave comment here about where this '20' value came from?

Reply:
Added a comment.